### DISCUSSION TITLE
Tilt effect on mouse hover

### DISCUSSION THREAD

**Discussion by pushmatrix** [google/model-viewer#2169] [Ideas]
I have been experimenting with a subtle tilt effect inspired by how Apple TV cards move [when you hover](https://codepen.io/arixking/pen/EWKzvR) over them.

![effect](https://user-images.githubusercontent.com/190692/58822380-57f2d000-8605-11e9-966b-bdd59b806167.gif)

Here's a quick prototype I threw together:
https://painted-sleep.glitch.me/

The camera tilts slightly based on where your mouse is hovered over the element. While it's a subtle effect, it helps reinforce how the model is 3D and not just an image. Adds an extra hint for discoverability.

Something to consider adding for that extra oomph!

**Comment by cdata** [comment]
That's awesome 😍 I love this idea as a subtle but intuitive improvement / alternative for our default interaction prompt.

I wonder if there is an additional version of this where the model tilts in line with the scroll value.. maybe that's a separate issue though :)

cc @elalish

**Comment by pushmatrix** [comment]
@cdata Took the words right out of my mouth! The tilt on scroll is something we've been playing around with too, and there is a great example of that type of interaction in the wild: https://panic.com/transmit/

I filed an issue for it here: #600

**Comment by raydeoactive** [comment]
Hey how do you add that mouse hover code?
Can you explain or share file please!

**Reply by pushmatrix** [reply]
@raydeoactive all code is in https://painted-sleep.glitch.me. Just view the page source and it's in the html.
